src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpNameReducer.cs

```diff
@@ -9,7 +9,6 @@
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Simplification;
 using Microsoft.CodeAnalysis.Text;
-using Roslyn.Utilities;
```

Comment:
note: we should probably rename this.  CSharpNameReducer isn't really valid.  But i don't have a better name.  It's like a huge amalgam of "type simplification, name simplification and expr simplification".  I would like things to eb broken out more, but it's unclear to me how to do that without a lot of churn.  There are *very* complex methods under hte covers that operate on SyntaxNode and do all sorts of casting and typechecks to decide what to do.

Reply:
These helpers are also available as extensions on core types like ExpressionSyntax, so touching things here is tricky.